README.md

```diff
@@ -583,6 +585,12 @@ See the `conda-forge`
 [jax](https://github.com/conda-forge/jax-feedstock#installing-jax) repositories
 for more details.
 
+### Containers
```

Comment:
Can we:
a) move this just below the other CUDA GPU subsections so all the CUDA pieces are together?
b) reword it. Perhaps:

```
### Docker containers: NVIDIA GPU

NVIDIA provides the [JAX Toolbox](...) containers, which are bleeding edge containers containing nightly releases of ...
```

Reply:
Done. I hope I moved where you wanted, not 100% sure.